Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json

```diff
@@ -0,0 +1,47 @@
+﻿{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/commit/3163b00, suitstorage sprites made by VictorJob",
```

Comment:
This needs to include the .dmi that the states were taken from, RSIEdit will do this automatically for you if you copy paste dmis into it
Example:
`Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/7cb618c69b75873f3ce893022fe08d1233b3152d/icons/obj/items/weapons/guns/attachments/rail.dmi`